plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/CachingJdbcClient.java

```diff
@@ -70,27 +71,37 @@
     private final JdbcClient delegate;
     private final List<PropertyMetadata<?>> sessionProperties;
     private final boolean cacheMissing;
+    private final JdbcIdentityCacheMapping identityMapping;
```

Comment:
nit: leave a blank line between immutable configuration and mutable state (caches)